intel-retail/performance-tools
# feat: enhance stream-density mode for the benchmark script to take multiple target_fps and container_names

this pr to add new capabilities for parallel running different case pipelines for stream density mode and added unit tests

## PR Checklist



## What are you changing?


## Issue this PR will close

close: #**issue_number**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

- git clone this PR
- change directory to **benchmark-scripts**: `cd ./benchmark-scripts`
- re-build the python packages:
```console
make init-packages
```
- run unit tests:
```console
make python-test
```
you should see all unit tests are PASSED and OK
- run coverage:
```console
make python-coverage
```
so that you can see the code coverage of the stream_density.py 

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/performance-tools )
